CONTRIBUTING.md

```diff
@@ -27,3 +29,29 @@ Do you have a very concrete proposal for an object type?
   and increased version number. **Then create a pull request.**
 
 Thank you for your efforts that will help to improve the CSMIM standard!
+
+
+
+## How is your contribution processed?
+
+![Process for changing the CSMIM knowledge base](docs/contribute-process.svg)
+
+1. The CSMIM working group discusses new issues internally and publicly on
+   Github with you. This serves to get a better understanding of your
+   requirements.
+
+2. If new object types are proposed as a result from the discussion, we will
+   create a branch and a pull request. We will attempt to gather more
+   industry experts for the topic at hand, in order to reach a good consensus
+   on the proposed interface. We will discuss and refine the proposed object
+   types with you in the pull request.
```

Comment:
better do the discussion in the issue and then only discuss the implementation in the pull request.